adobe/spectrum-css
# fix(button -> negative): corrects interior label background color to align with parent

## Description

- Apples forced-color-adjust: none to align the rendered background of the child label for negative button varient with the enclosing button's background color.

## How and where has this been tested?

Verified in Assistiv Labs high contrast mode-enabled Windows VM.

### Validation steps

1. Run Storybook locally (or reference the link for this PR).
2. Navigate to Assistiv Labs and start a Windows High Contrast Mode VM (if running the branch locally, you'll need to enable the Assistiv Labs tunnel to access Storybook).
3. Enable Windows High Contrast Mode.
4. Navigate to the Button component in Storybook.
5. Select the `negative` variant.
6. Zoom in and verify that the there is no background (the button background should be a uniform color).

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:


7. If components have been modified, VRTs have been run on this branch:


## Screenshots

<img width="1561" alt="Screenshot 2024-09-12 at 4 46 58 PM" src="https://github.com/user-attachments/assets/98ecdb9d-a914-4589-be96-2bd0e717d7d6">
<img width="1569" alt="Screenshot 2024-09-12 at 4 46 12 PM" src="https://github.com/user-attachments/assets/7df930c0-c89b-4c78-8659-e75eee95a32f">

## To-do list
